src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -116,6 +138,8 @@ else if (o instanceof Slot)
 				case REMOVE_ALL:
 					assert false;
 			}
+			if (durability && itemStack != null && mode != ChangeMode.RESET && mode != ChangeMode.DELETE)
+				changeValue = itemStack.getType().getMaxDurability() - changeValue;
 			if (o instanceof ItemType && itemStack != null) {
 				ItemUtils.setDamage(itemStack,changeValue);
```

Comment:
While you're here, can you add a space to this param and same for other params below it
```suggestion
				ItemUtils.setDamage(itemStack, changeValue);
```